dashboard/db/migrate/20170502232412_associate_course_and_plc_course.rb

```diff
@@ -0,0 +1,8 @@
+class AssociateCourseAndPlcCourse < ActiveRecord::Migration[5.0]
+  def up
+    Plc::Course.all.each do |plc_course|
+      course = Course.create(name: plc_course.name, plc_course_id: plc_course.id)
```

Comment:
Am I right in thinking we don't need to reload the column information (see, e.g., [here](http://stackoverflow.com/questions/9115347/model-reset-column-information-does-not-reload-columns-in-rails-migration)) because the `add_column` was in a previous migration? What is the lifetime of the column cache?